components/Buttons/examples/ButtonsShapesExampleViewController.m

```diff
@@ -80,7 +91,7 @@ - (void)viewDidLoad {
 
   MDCButton *disabledContainedButton = [[MDCButton alloc] init];
   [disabledContainedButton setTitle:@"Disabled" forState:UIControlStateNormal];
-  [MDCContainedButtonThemer applyScheme:buttonScheme toButton:disabledContainedButton];
+  [disabledContainedButton applyContainedThemeWithScheme:self.containerScheme];
```

Comment:
Since containerScheme property isn't explicitly defined, maybe using `[self containerScheme]` here to keep up with the change above.